tests/roots/test-latex-table/expects/complex_spanning_cell.tex

```diff
@@ -1,10 +1,10 @@
-\label{\detokenize{complex:complex-spanning-cell}}
+\label{\detokenize{complex:complex-spanning-cell}}\hskip0pt\relax
 table having …
 \begin{itemize}
-\item {} 
+\item {} \hskip0pt\relax
```

Comment:
I thought about modifiying `visit_list_item` but as the item might be a nested sublist, the insertion must come from actual paragraph. Notice here that the blanks in the sequence ` {} \hskip0pt\relax` introduces no extra space at all in pdf output.

Reply:
Now `\hskip0pt\relax` is modified into `\sphinxAtStartPar`, and here it would be on next line so the `\item {}<space>` line remains exactly as in current Sphinx. So this will not modify who parse output file for this specific line for whatever purpose.